packages/forms-web-app/src/views/pages/examination/select-file.njk

```diff
@@ -154,8 +154,8 @@
 				{% endif %}
 			>
 				const target = document.documentElement;
-				const dropZone = document.querySelector('.dropZone');
-				const fileInput = document.getElementById('file-upload')
+				const dropZone = document.querySelector(".dropZone");
+				const fileInput = document.getElementById("file-upload");
```

Comment:
Quotes correcting to double? Is this right? I think mine are single.

Reply:
passing linter ?

Reply:
not a JS file - so prob no linting 🤔

Reply:
'fixed'